server/pulp/server/webservices/views/serializers/content.py

```diff
@@ -1,23 +1,10 @@
-# -*- coding: utf-8 -*-
-#
-# Copyright © 2011 Red Hat, Inc.
-#
-# This software is licensed to you under the GNU General Public
-# License as published by the Free Software Foundation; either version
-# 2 of the License (GPLv2) or (at your option) any later version.
-# There is NO WARRANTY for this software, express or implied,
-# including the implied warranties of MERCHANTABILITY,
-# NON-INFRINGEMENT, or FITNESS FOR A PARTICULAR PURPOSE. You should
-# have received a copy of GPLv2 along with this software; if not, see
-# http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
-
 """
 Module for content serialization.
 """
 
 from pulp.common import dateutils
 from pulp.server.webservices import http
-from pulp.server.webservices.serialization import db
+from pulp.server.webservices.views.serializers import db
 
 
 # constants --------------------------------------------------------------------
```

Comment:
I know you didn't write these, but I remove them whenever possible. PEP8 provides conventions which makes these section dividing comments unnecessary.